src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
According to the Code Style Rules, styles should be linked with a .css extension, not .scss. Please change the 'href' attribute to point to a .css file.